### PR TITLE
Add Windows instructions for Jekyll to website and README

### LINKED ISSUE #761: [DOC] - Windows instructions for website contributions via Jekyll
1) There are no Windows instructions for Jekyll at https://vmweventbroker.io/kb/contribute-site , though there are Windows instructions in /docs/README.md. The README instructions need to be included on the webpage

2) Windows can throw warnings saying `Auto-regeneration may not work on some Windows versions`, which forces you to restart the container on every change. This makes website contributions time consuming. Workaround is the --force-polling argument to jekyll serve.

Update the webpage and README accordingly.

### PR DESCRIPTION
**Pull request by kremerpatrick**
Signed-off-by: Patrick Kremer <pkremer@vmware.com>

## Summary

Added Windows instructions to the website - previously they only existed in the README.
Added --force-polling workaround to Windows instructions

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [ ] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [ ] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [ ] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commits related to your change
- [ ] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [ ] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation changes
- [ ] Other (please describe):

## Resolved Issues

Closes #761 

## Testing Verification
 
Built locally, tested in browser.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -78,6 +78,14 @@ docker run --rm `
   jekyll/jekyll `
   jekyll serve
 ```
+> Note: You may seen a warning saying `Auto-regeneration may not work on some Windows versions.` Symptoms of this warning are the inability to automatically see any local website changes in your browser. This makes website changes difficult as you must restart Jekyll after every local change to view the results. To work around this issue, try the `--force-polling` switch to the `jekyll serve` command.
```
Comment (embano1) [inline docs/README.md:81]: nit: can you please line-wrap such long comments? there's shortcuts/tools/plugins for every major IDE, e.g. Option+Q in vscode.

File: docs/README.md [hunk 2]
```diff
@@ -78,6 +78,14 @@ docker run --rm `
...
+```powershell
+docker run --rm `
+  --volume="${PWD}:/srv/jekyll" `
+  --publish 4000:4000 `
+  jekyll/jekyll `
```
Comment (embano1) [inline docs/README.md:86]: should we specify a tag here? see linux/mac section

**Review by embano1 — CHANGES_REQUESTED** [review]
Please update the commit: the title is too long and the closes comment does not belong in the title but commit body.

File: docs/README.md (hunk 1 above)
Comment (kremerpatrick) [reply docs/README.md:81]: Opt+Q doesn't seem to work out of the box, but there's an extension called "Rewrap" that uses Option+Q and seems to do what you're asking for.

File: docs/README.md (hunk 2 above)
Comment (kremerpatrick) [reply docs/README.md:86]: Fixed

File: docs/README.md (hunk 1 above)
Comment (embano1) [reply docs/README.md]: Ah, wasn't sure whether it's builtin or extension. Cool!

**Comment by embano1** [comment]
Thx, happy to merge but the commit doesn't have `Closes: #761` in the body so it won't correctly link your issue in the release notes. Let me know if you want to change that or I merge.

File: docs/kb/contribute-site.md [hunk 3]
```diff
@@ -76,6 +77,24 @@ docker run --rm \
...
+  jekyll/jekyll:3.8 `
+  jekyll serve
+```
+
+> Note: You may seen a warning saying `Auto-regeneration may not work on some Windows versions.` Symptoms of this warning are the inability to automatically see any local website changes in your browser. This makes website changes difficult as you must restart Jekyll after every local change to view the results. To work around this issue, try the `--force-polling` switch to the `jekyll serve` command.
+```powershell
+docker run --rm `
+  --volume="${PWD}:/srv/jekyll" `
+  --publish 4000:4000 `
+  jekyll/jekyll `
```
Comment (embano1) [inline docs/kb/contribute-site.md:94]: nit: tag is missing

Comment (kremerpatrick) [reply docs/kb/contribute-site.md]: Fixed

**Comment by kremerpatrick** [comment]
I added the Closes line to the commit comment.

**Review by embano1 — APPROVED** [review]

**Pull request merged**